public/js/poll/view.js

```diff
@@ -86,9 +86,11 @@
 			poll = View.parseResults(poll);
 			for (var i = 0, l = poll.options.length; i < l; i++) {
 				var option = poll.options[i],
-					optionView = pollView.find('#pollResult' + option.id);
+				    optionView = pollView.find('.poll-view-result[data-poll-result="'+option.id+'"]');
```

Comment:
This I think is simply a 0.7.x compatibility issue but again the anchor was incorrect, ```#pollResult``` was nowhere to be found.